<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>668</width>
    <height>540</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Thêm câu trả lời đánh giá</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(40, 44, 52);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QWidget{
color: rgb(210, 210, 210);
}
QPlainTextEdit {
	background-color: rgb(39, 44, 54);
	border-radius: 5px;
	border: 2px solid rgb(27, 29, 35);
	padding-left: 10px;
}
QPlainTextEdit:hover {
	border: 2px solid rgb(64, 71, 88);
}
QPlainTextEdit:focus {
	border: 2px solid rgb(91, 101, 124);
}

/* COMBOBOX */
QComboBox{
	background-color: rgb(27, 29, 35);
	border-radius: 5px;
	border: 2px solid rgb(27, 29, 35);
	padding: 5px;
	padding-left: 10px;
}
QComboBox:hover{
	border: 2px solid rgb(64, 71, 88);
}
QComboBox::drop-down {
	subcontrol-origin: padding;
	subcontrol-position: top right;
	width: 40px; 
	border-left-width: 3px;
	border-left-color: rgba(39, 44, 54, 150);
	border-left-style: solid;
	border-top-right-radius: 3px;
	border-bottom-right-radius: 3px;	
	background-image: url(:/icon/cil-chevron-bottom.png);
	background-position: center;
	background-repeat: no-reperat;
 }
QComboBox QAbstractItemView {
	color: rgb(85, 170, 255);	
	background-color: rgb(27, 29, 35);
	padding: 10px;
	selection-background-color: rgb(39, 44, 54);
}
</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QComboBox" name="comboBox_ratting">
         <property name="font">
          <font>
           <family>Nunito</family>
           <pointsize>17</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QComboBox{
	background-color: rgb(27, 29, 35);
	border-radius: 5px;
	border: 2px solid rgb(27, 29, 35);
	padding: 5px;
	padding-left: 10px;
}
QComboBox:hover{
	border: 2px solid rgb(64, 71, 88);
}
QComboBox QAbstractItemView {
	color: rgb(85, 170, 255);	
	background-color: rgb(27, 29, 35);
	padding: 10px;
	selection-background-color: rgb(39, 44, 54);
}</string>
         </property>
         <item>
          <property name="text">
           <string>Đánh giá 1 sao</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Đánh giá 2 sao</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Đánh giá 3 sao</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Đánh giá 4 sao</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Đánh giá 5 sao</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="plainTextEdit_ratting">
         <property name="font">
          <font>
           <family>Nunito</family>
           <pointsize>18</pointsize>
          </font>
         </property>
         <property name="inputMethodHints">
          <set>Qt::ImhNone</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_send">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>50</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Nunito</family>
           <pointsize>24</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">#btn_send{
	border-radius: 25px;
    padding:8px 25px 7px 25px;
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(89, 64, 231, 255), stop:1 rgba(15, 181, 253, 255));
	color: rgb(255, 255, 255);

}

#btn_send:hover{
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(15, 181, 253, 255), stop:1 rgba(89, 64, 231, 255));
}

#btn_send:pressed{
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(89, 64, 231, 255), stop:1 rgba(15, 181, 253, 255));
}</string>
         </property>
         <property name="text">
          <string>Thêm câu trả lời</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../qrc/file_img.qrc"/>
 </resources>
 <connections/>
</ui>
